src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstallerProjectEvents.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.Composition;
+using NuGet.ProjectManagement;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains batch events to be raised when performing multiple packages install/ uninstall in a project
+    /// </summary>
+    [Export(typeof(IVsPackageInstallerProjectEvents))]
+    [Export(typeof(VsPackageInstallerProjectEvents))]
```

Comment:
I know only interface is needed but then existing IVsPackageInstallerEvents has class import as well so i thought this is expected. But I'll remove it.